prometheus-hanadb_exporter.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Tue Feb 11 10:45:43 UTC 2020 - Dario Maiocchi <dmaiocchi@suse.com>
+
+- Version 0.6.2: Rename service systemd to prometheus prefix (jsc#SLE-10545)
```

Comment:
Remove the `:` character after the version number. The other entries don't have it. Just for consistency